apps/src/templates/Button.jsx

```diff
@@ -114,10 +115,19 @@ class Button extends React.Component {
     // potential exploits. Therefore, we do so here.
     const rel = target === '_blank' ? 'noopener noreferrer' : undefined;
 
+    let tagStyle;
+    let className = this.props.className || '';
+    if (displayAsText) {
+      tagStyle = [styles.main, styles.textButton, style];
+      className += 'button-active-no-border';
```

Comment:
does this still work if `this.props.className` is provided? this class would end up as `className="my-classbutton-active-no-border"`

Reply:
Oh excellent catch! You're right, I'll replace this with using the classnames library to concat them